app/views/staff/funds/new.html.haml

```diff
@@ -0,0 +1,19 @@
+=content_for :page_title_prefix, t("page_title.fund.new")
+
+%p
+  = link_to t("generic.link.back"), funds_path, class: "govuk-back-link"
+%main.govuk-main-wrapper#main-content{ role: "main" }
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      %h1.govuk-heading-xl
+        = t("page_title.fund.new")
+
+      = simple_form_for @fund do |f|
+        .govuk-form-group
+          = f.input :organisation_id,
```

Comment:
When my user doesn't have an organisation I get an empty select box and no call to action to fix it. Users don't require an organisation at the moment because of administrators. Perhaps it would be simpler to change that now so that admins always belong to BEIS?

If we don't do the above, then it might make sense to hide the form and show a call to action message insteaed. Which might be that they need to be invited to at least one organisation first and to contact a memeber of the BEIS team.

Either way we should have test coverage for this.